drivers/hmis/app/graphql/types/hmis_schema/file.rb

```diff
@@ -52,7 +52,7 @@ def updated_by
     end
 
     def own_file
-      object.user&.user_id == current_user.id.to_s
+      object.uploaded_by_id == Hmis::Hud::User.from_user(current_user).id
```

Comment:
Hmm, I'm not sure we want the `Hmis::Hud::User`.  `current_user` should suffice for files.  `Hmis::Hud::User` would be referenced in the standard HMIS tables.